File: TEMA3/typescript/ordenacion.ts
function ordenar(lista: number[], ascendente: boolean)
{
    lista = ascendente == false ? lista.toSorted() : lista.toReversed();
    return lista;
}

const lista: number[] = [7, 12, 9, 11, 3]
console.log("Lista ascendente")
console.log(ordenar(lista, true))
console.log("Lista descendente")
console.log(ordenar(lista, false))

File: TEMA3/python/ordenacion.py
def ordenar(lista: list, ascendente=True):
    if ascendente:
        return sorted(lista)
    else:
        return sorted(lista, reverse=True)

def ordenar2(lista: list, ascendente=True):
    if ascendente:
        lista.sort()   #
    else:
        lista.reverse()   #lista.sort(reverse=True)
    return lista


list = [7, 12, 9, 11, 3]

print(ordenar2(list))
print(ordenar2(list, False))

